<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_task_window</class>
 <widget class="QWidget" name="edit_task_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(17, 15, 27);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameFile">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	
	background-color: rgb(235, 235, 235);
	border-color: rgb(196, 196, 196);
	color: rgb(119, 120, 115);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textFile">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit
{
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	
	background-color: rgb(235, 235, 235);
	border-color: rgb(196, 196, 196);
	color: rgb(119, 120, 115);
}

    QScrollBar:vertical {
        background: #e0e0e0;
        width: 12px;
        margin: 2px 0 2px 0;
        border-radius: 6px;
    }
    QScrollBar::handle:vertical {
        background: #9c9c9c;
        border-radius: 6px;
        min-height: 20px;
    }
    QScrollBar::handle:vertical:hover {
        background: #666;
    }
    QScrollBar::add-line:vertical,
    QScrollBar::sub-line:vertical {
        height: 0;
    }
    QScrollBar::add-page:vertical,
    QScrollBar::sub-page:vertical {
        background: none;
    }</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Змінити дату:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dataFile">
       <property name="styleSheet">
        <string notr="true">QDateEdit
{
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	background-color: rgb(235, 235, 235);
	border-color: rgb(196, 196, 196);
	color: rgb(119, 120, 115);
}
QDateEdit::up-button
{
	border-style: outset;
    border-width: 2px;
    border-radius: 4px;

	border:none;
	background-color: rgb(196, 196, 196);
}
QDateEdit::down-button
{
	border-style: outset;
    border-width: 2px;
    border-radius: 4px;

	border:none;
	background-color: rgb(196, 196, 196);
}
QDateEdit::up-arrow {
        image: url(:/icon/icons/up_arrow.png);
        width: 20px;
        height: 10px;
    }
QDateEdit::down-arrow {
        image: url(:/icon/icons/down_arrow.png);
        width: 20px;
        height: 10px;
    }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font-size: 12pt;
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	background-color: rgb(152, 133, 248);
	color: rgb(0, 0, 0);
	border-color: rgb(17, 15, 27);
}
QPushButton:hover
{
	background-color: rgb(121, 97, 245);
}
QPushButton:pressed
{
	background-color: rgb(97, 72, 226);
}</string>
       </property>
       <property name="text">
        <string>Скасувати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doneBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font-size: 12pt;
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	background-color: rgb(152, 133, 248);
	color: rgb(0, 0, 0);
	border-color: rgb(17, 15, 27);
}
QPushButton:hover
{
	background-color: rgb(121, 97, 245);
}
QPushButton:pressed
{
	background-color: rgb(97, 72, 226);
}
</string>
       </property>
       <property name="text">
        <string>Примінити зміни</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
